File: ws/sisalema/u1/ws10Collections/Collections_Python_Javascript/Person.py
class Person:
    def __init__(self, id, name):
        self._id = id
        self._name = name

    def __str__(self):
        return f"Person{{id={self._id}, name='{self._name}'}}"

    @property
    def id(self):
        return self._id

    @id.setter
    def id(self, id):
        self._id = id

    @property
    def name(self):
        return self._name

    @name.setter
    def name(self, name):
        self._name = name

File: ws/sisalema/u1/ws10Collections/Collections_Python_Javascript/Person.js
<<<<<<< HEAD
=======
<<<<<<< HEAD
>>>>>>> a064af6ecab12755c8c8b8a5858b342fd47aa80f
export class Person {
    constructor(id, name) {
        this._id = id;
        this._name = name;
    }

    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get name() {
        return this._name;
    }

    set name(name) {
        this._name = name;
    }

    toString() {
        return `Person{id: ${this._id}, name: '${this._name}'}`;
    }
<<<<<<< HEAD
=======
=======
export class Person {
    constructor(id, name) {
        this._id = id;
        this._name = name;
    }

    get id() {
        return this._id;
    }

    set id(id) {
        this._id = id;
    }

    get name() {
        return this._name;
    }

    set name(name) {
        this._name = name;
    }

    toString() {
        return `Person{id: ${this._id}, name: '${this._name}'}`;
    }
>>>>>>> 8e40484f0077d7fe10786aa1aa0f3cc51debe068
>>>>>>> a064af6ecab12755c8c8b8a5858b342fd47aa80f
}